src/components/createTransaction/createTransaction.tsx

```diff
@@ -0,0 +1,126 @@
+import { Transaction, TransactionType } from "@blockr/blockr-models";
```

Comment:
Your file naming is inconsistent within this folder and potentially in other folders as well.